DataFormats/interface/DoubleHTag.h

```diff
@@ -97,14 +101,17 @@ namespace flashgg {
         float phidipho() const {return phidipho_;}
         float fabs_CosThetaStar_CS() const {return fabs_CosThetaStar_CS_;}
         float fabs_CosTheta_bb() const {return fabs_CosTheta_bb_;}
-
+        float mass_corr() const {return mass_corr_;}
 
     private:
         double mva_, MX_, genMhh_,genCosThetaStar_CS_;
         vector<float> benchmark_reweights_;
  //       std::vector<float> mva_prob_;
          long eventNumber_;
         edm::Ptr<flashgg::Jet> leadJet_, subleadJet_;
+        double* RegMET_;
```

Comment:
why are these variables pointers?

Reply:
I didn't understand your question. I assigned as pointer to pass by address of variable which are used to calculate Mjj regression

Reply:
using bare pointers like this can be very error prone, just use variables and add a method to return a pointer to the variable when called instead (if you really need a pointer anywhere at all)

Reply:
Thanks. I have corrected it now.